schemas/data/metrics.schema.json

```diff
@@ -31,12 +31,36 @@
         "https://ns.adobe.com/xdm/data/metrics/advertising/clicks": {
           "$ref": "https://ns.adobe.com/xdm/data/measure"
         },
+        "https://ns.adobe.com/xdm/data/metrics/application/closes": {
```

Comment:
Where are these application things coming from?  Is there some standard here - even within Adobe?  Does it come from a specific SDK?

Reply:
@lrosenthol  The primary library that is informing this set of additions in the Appmeasurement library for mobile (iOS/Android) with has been around for many years and is well adopted in our customer base. The metrics being added for applications are common across the industry for mobile applications.
Also there are a number of metrics tracked by creative cloud and doc cloud desktop applications when end users enroll in providing Adobe with this data. Most of the standard metrics line up well, the addition is featureUsage. 
One open item I am tracking is the concept of a installer download for desktop. More research is needed there to see if it aligns with a regular content download or is an application download is a different concept entirely.

Reply:
Thanks for the info!